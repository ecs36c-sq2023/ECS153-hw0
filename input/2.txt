Student
5 
b 3
a 3
c 3
Ada 1
Peter 2
