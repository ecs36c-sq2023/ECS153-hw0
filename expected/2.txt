c 3
b 3
a 3
Peter 2
Ada 1
